packages/resizable-mixin/resizable-mixin.ts

```diff
@@ -0,0 +1,68 @@
+import { LitElement } from 'lit-element';
+import getResizeObserver from './resize-observer-polyfill';
+import { ResizableClass } from './resizable-class';
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+type Constructor<T = object> = new (...args: any[]) => T;
+
+const resizables = new Set();
+
+export const ResizableMixin = <T extends Constructor<LitElement>>(
+  base: T
+): T & Constructor<ResizableClass> => {
+  class Resizable extends base {
+    protected static _resizeObserver?: ResizeObserver;
+
+    connectedCallback() {
+      super.connectedCallback();
+
+      resizables.add(this);
+
+      this._initResizeObserver().then(() => {
+        const observer = Resizable._resizeObserver;
+        observer && observer.observe(this);
+        resizables.add(this);
```

Comment:
Is this also needed while this was already added to `resizables` synchronously in connectedCallback?

Reply:
Removed.